lib/lm_docstache/hide_custom_tags.rb

```diff
@@ -0,0 +1,67 @@
+module LMDocstache
+  class HideCustomTags
+    HIDE_BACKGROUND_COLOR = 'FFFFFF'
+
+    attr_reader :document, :hide_custom_tags
+    def initialize(document:, hide_custom_tags: [])
+      @document = document
+      @hide_custom_tags = hide_custom_tags
+    end
+
+    # Find all nodes matching hide custom tags and
+    # replace background to white.
```

Comment:
It's not the background color, it's the foreground color (font color). And what do you mean with "hide custom tags"? It'd be nice to explain the rationale that outlines the purpose of this method/class, because it's still not very clear with this comment.